### PR TITLE
changes to service definition, README, and added new target to Makefile

### PR DESCRIPTION
**Pull request by t-fine**
- Added here target for icp startup script
- changed service name from cpu to ibm.cpu 
- made those changes in README and port changed to 80

### CODE REVIEW

File: edge/msghub/cpu2msghub/Dockerfile.amd64 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 FROM alpine:latest
-RUN apk --no-cache --update add curl ca-certificates wget jq bc
+RUN apk --no-cache --update add curl ca-certificates wget jq bc bash
```
Comment (bmpotter) [inline edge/msghub/cpu2msghub/Dockerfile.amd64]: Why did you add bash? I'm not seeing other changes that require it. An edge service best practice is to keep the images as small as possible.

File: edge/msghub/cpu2msghub/Makefile [hunk 2]
```diff
@@ -58,6 +58,12 @@ publish-all-arches:
 	ARCH=arm64 $(MAKE) build-test-publish
 	hzn exchange pattern publish -f horizon/pattern-all-arches.json
 
+icp-publish:
```
Comment (bmpotter) [inline edge/msghub/cpu2msghub/Makefile]: Sorry i didn't mention this before, but i think we will end up needing this target on the ibm public cloud too. Can you change the name of the target to something like publish-only ?

File: edge/msghub/cpu2msghub/horizon/service.definition.json [hunk 3]
```diff
@@ -27,13 +27,13 @@
             "name": "MSGHUB_API_KEY",
             "label": "The API key to use when sending messages to your instance of IBM Message Hub",
             "type": "string",
-            "defaultValue": ""
+            "defaultValue": "$MSGHUB_API_KEY"
```
Comment (bmpotter) [inline edge/msghub/cpu2msghub/horizon/service.definition.json]: Where is the `$MSGHUB_API_KEY` variable coming from? This is information that is only known by the node owner or the organization, which is why the default value was left blank.

File: edge/msghub/cpu2msghub/horizon/service.definition.json [hunk 4]
```diff
@@ -27,13 +27,13 @@
...
         },
         {
             "name": "MSGHUB_BROKER_URL",
             "label": "The comma-separated list of URLs to use when sending messages to your instance of IBM Message Hub",
             "type": "string",
-            "defaultValue": ""
+            "defaultValue": "$MSGHUB_BROKER_URL"
```
Comment (bmpotter) [inline edge/msghub/cpu2msghub/horizon/service.definition.json]: same comment as above

File: edge/msghub/cpu2msghub/Dockerfile.amd64 (hunk 1 above)
Comment (t-fine) [reply edge/msghub/cpu2msghub/Dockerfile.amd64]: Oh shoot, I forgot to remove that. I added it to test with the new gps so I could exec into the cpu2msghub container and curl gps manually. I'll get rid of it!

File: edge/msghub/cpu2msghub/Makefile (hunk 2 above)
Comment (t-fine) [reply edge/msghub/cpu2msghub/Makefile]: Will do!

File: edge/msghub/cpu2msghub/horizon/service.definition.json (hunk 3 above)
Comment (t-fine) [reply edge/msghub/cpu2msghub/horizon/service.definition.json]: I think in moving some of those variables over I copied and pasted these lines and probably forgot to remove it.
